server/conference-data/test.js

```diff
@@ -1,5 +1,7 @@
+/* eslint-disable max-len */
```

Comment:
Don't disable the linter. Instead fix the style error.

No point in having a linter if we turn it off.